### PR TITLE
Add new connection types to the taxonomy layer in connection.yaml.

### PR DESCRIPTION
**Pull request by revit13**
Related to #1866 

This PR adds new connection types to the taxonomy layer in connection.yaml.

### CODE REVIEW

**Review by shlomitk1 — CHANGES_REQUESTED** [review]
You need to submit the generated files as well.

**Comment by revit13** [comment]
> You need to submit the generated files as well.

@shlomitk1 Fixed thanks

**Comment by shlomitk1** [comment]
> > You need to submit the generated files as well.
> 
> @shlomitk1 Fixed thanks

Also documentation files

**Comment by revit13** [comment]
> > > You need to submit the generated files as well.
> > 
> > 
> > @shlomitk1 Fixed thanks
> 
> Also documentation files

Fixed Thanks

**Review by shlomitk1 — CHANGES_REQUESTED** [review]
Models/mysql.md is not submitted.

**Comment by revit13** [comment]
> Models/mysql.md is not submitted.

Fixed. Thanks

**Review by shlomitk1 — APPROVED** [review]

File: charts/fybrik/files/taxonomy/taxonomy.json [hunk 1]
```diff
@@ -294,6 +300,19 @@
...
+      "properties": {
+        "url": {
+          "type": "string",
+          "description": "The URL path to access the file."
+        }
+      },
+      "required": [
+        "url"
+      ]
+    },
```
Comment (shlomitk1) [inline charts/fybrik/files/taxonomy/taxonomy.json:315]: Can this connection be used for storing data? Does it make sense to define a storage account for this type? If so, perhaps to define more fields, like domain/base_url, folder and file?

File: charts/fybrik/files/taxonomy/taxonomy.json [hunk 2]
```diff
@@ -338,6 +357,19 @@
...
+      "type": "object",
+      "description": "Connection information for accessing data in local filesystem",
+      "properties": {
+        "destination_path": {
+          "type": "string",
+          "description": "Path to the directory where the data is stored"
+        }
+      },
+      "required": [
+        "destination_path"
```
Comment (shlomitk1) [inline charts/fybrik/files/taxonomy/taxonomy.json:370]: What does this path include? Can it be split into more details, to be able to generate a connection based on pre-exisitng information for storage allocation?

File: charts/fybrik/files/taxonomy/taxonomy.json (hunk 1 above)
Comment (revit13) [reply charts/fybrik/files/taxonomy/taxonomy.json:309]: @shlomitk1 rethinking about that, there are libs such as `url.Parse` which could parse the url and provide the fields you mentioned, so we do not need to specify them explicitly but rather provide the url. Does it make sense?

Comment (shlomitk1) [reply charts/fybrik/files/taxonomy/taxonomy.json:309]: ok

File: site/external.yaml [hunk 3]
```diff
@@ -1 +1 @@
-FybrikRelease:  __Release__
+FybrikRelease:  v1.1.1
```
Comment (shlomitk1) [inline site/external.yaml:1]: Why 1.1.1?

Comment (revit13) [reply site/external.yaml]: Leftover from testing another PR. Thanks for the catch!

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**